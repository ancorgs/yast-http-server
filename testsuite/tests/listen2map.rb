# encoding: utf-8

#testfile: routines.ycp
#return list of maps [ address:"address", port:"port" ]
module Yast
  class Listen2mapClient < Client
    def main
      Yast.import "UI"
      Yast.include self, "testsuite.rb"


      Yast.include self, "http-server/routines.rb"

      TEST(lambda { listen2map("192.168.1.2:80") }, [{}, {}, {}], nil)

      nil
    end

    def Modified
      true
    end

    def AbortFunction
      true
    end
  end
end

Yast::Listen2mapClient.new.main
