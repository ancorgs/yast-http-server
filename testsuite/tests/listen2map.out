Read	.target.tmpdir nil
Log	Failed to set temporary directory: nil
Return	$["address":"192.168.1.2", "port":"80"]
